R/deployApp.R

```diff
@@ -151,6 +158,7 @@ deployApp <- function(appDir = getwd(),
                       appSourceDoc = NULL,
                       appName = NULL,
                       appTitle = NULL,
+                      envVars = NULL,
```

Comment:
Any reason why `envVars` is not the final argument? Placing it here breaks any calls which assume argument order.

Reply:
In a function like this, I think it's reasonable to assume that folks will name arguments rather than relying on position (especially at position 7), and this allows us to make the importance of this variable more obvious by listing it earlier in the function signature.

If you're worried about this silently changing existing code we code at `...` to the function signature (forcing all argument after the `...` to be named) and use `rlang::check_dots_empty()` to give an informative error.